src/backend/src/prisma/schema.prisma

```diff
@@ -331,37 +334,49 @@ model LinkType {
 }
 
 model Link {
-  linkId       String      @id @default(uuid())
-  url          String
-  creatorId    Int
-  creator      User        @relation(name: "linkCreator", fields: [creatorId], references: [userId])
-  dateCreated  DateTime    @default(now())
-  dateDeleted  DateTime?
-  linkTypeName String
-  linkType     LinkType    @relation(name: "linkTypes", fields: [linkTypeName], references: [name])
-  wbsElementId Int
-  wbsElment    WBS_Element @relation(name: "links", fields: [wbsElementId], references: [wbsElementId])
+  linkId              String                @id @default(uuid())
+  url                 String
+  creatorId           Int
+  creator             User                  @relation(name: "linkCreator", fields: [creatorId], references: [userId])
+  dateCreated         DateTime              @default(now())
+  dateDeleted         DateTime?
+  linkTypeName        String
+  linkType            LinkType              @relation(name: "linkTypes", fields: [linkTypeName], references: [name])
+  wbsElementId        Int
+  wbsElment           WBS_Element           @relation(name: "links", fields: [wbsElementId], references: [wbsElementId])
```

Comment:
why would it be LinkTypeInfo and not LinkInfo if we're basing the new model off the link model?

Reply:
To clarify we're making a model for links that doesn't contain the wbs element stuff because for proposed changes we don't want them to have associated wbs elements since the link model has wbs elements we need to create a new one that resembles it but doesn't have the wbs element stuff

Reply:
Then in the proposed changes I'd use the linkInfo model instead of the link model

Reply:
Yup exactly

Reply:
would the blocked by also need to be changed since it has wbs element